client/src/pages/ItemsList.js

```diff
@@ -1,240 +1,213 @@
+  }
+
+  fetchAllItems = () => {
+    api
+      .getAllItems()
+      .then(resp => {
+        const { items } = resp.data;
+        console.log('getAllItems: resp');
         console.log(items);
+        this.setState({ items });
+      })
+      .catch(err => {
+        console.error(`ERROR in 'getAllItems': ${err}`);
+        console.error(err);
+        return err;
+      });
+  };
+
+  deleteSingleItem = itemId => {
+    return api
+      .deleteItemById(itemId)
+      .then(resp => {
+        console.log('deleteItemById: resp');
+        console.log(resp);
+        return resp;
+      })
+      .catch(err => {
+        console.error(`ERROR in 'deleteSingleItem': ${err}`);
+        console.error(err);
+        return err;
+      });
+  };
+
+  handleRemoveItem = data => {
+    const itemId = data;
+
+    this.deleteSingleItem(itemId).then(resp => {
+      console.log('handleRemoveItem: resp');
+      console.log(resp);
+      this.fetchAllItems();
+    });
+  };
+
+  render() {
+    const items = this.state.items || {};
+    console.log(items);
+
+    const columns = [
+      {
+        Header: 'ID',
+        accessor: '_id',
+        // filterable: true,
+        Cell: props => {
+          console.log(props);
+          const { original } = props.cell.row;
+          return <span data-item-id={original._id}>{props.value}</span>;
+        },
+      },
+      {
+        Header: 'Name',
+        accessor: 'name',
+        // filterable: true,
+        Cell: props => {
+          const { original } = props.cell.row;
+          return <span data-name={original.name}>{props.value}</span>;
+        },
+      },
+      {
+        Header: 'Day(s)',
+        accessor: 'daysOfWeek',
+        // filterable: true,
+        Cell: props => {
+          const { daysOfWeek } = props.cell.row.original;
+          let daysToDisplay = '';
+          if (daysOfWeek && typeof daysOfWeek === 'object') {
+            for (const day in daysOfWeek) {
+              daysToDisplay =
+                daysToDisplay === '' ? daysOfWeek[day] : `${daysToDisplay}, ${daysOfWeek[day]}`;
+            }
+          }
+          return (
+            <span
+              data-daysofweek={daysOfWeek && JSON.stringify(daysOfWeek)}
+              data-daysofweek-by-id={props.value}>
+              {daysToDisplay || '-'}
+            </span>
+          );
+        },
+      },
+      {
+        Header: 'Timeframe',
+        accessor: 'timeframeNote',
+        Cell: props => {
+          const { original } = props.cell.row;
+          return <span data-timeframe={original.timeframeNote}>{props.value || '-'}</span>;
+        },
+      },
+      {
+        Header: 'Priority',
+        accessor: 'priority',
+        // filterable: true,
+        Cell: props => {
+          const { original } = props.cell.row;
+          return <span data-priority={original.priority}>{props.value}</span>;
+        },
+      },
+      {
+        Header: '',
+        accessor: 'd',
```

Comment:
Shouldn't this be the following?
```suggestion
        Header: 'Update',
        accessor: '_update',
```

Reply:
Good catch